animations/transition_widgets.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: "Flutter transition animations"
+permalink: /animations/transitions_widgets.html
+---
+
+* TOC Placeholder
+{:toc}
+
+Transition animations implement the abstract class `AnimatedWidget`. The `AnimatedWidget` class allows you to separate the widget code from the animation code in the `setState()` call. The `AnimatedWidget` class doesn’t need to maintain a `State` object to hold the animation. Use the  `AnimatedWidget` helper class (instead of `addListener()` and `setState()`) to create a reusable transition animation widget.
+
+< additional info on transition widgets >
+
+
+## Using a transition animation widget
+
+<div>
+<table class="table" width="100%">
+  <col width="35%">
+  <col width="65%">
+	<tbody>
+    <tr>
+      <td><img src="images/transitions.png" alt="Transition animation widgets"></td>
```

Comment:
1. I like the idea of this list, but make it more of a general description without so much implementation detail.
2. Then link pretty quickly to an example for those who just want to see code.